nautobot/docs/overview/application_stack.md

```diff
@@ -0,0 +1,96 @@
+# Application Stack
+
+Nautobot is built on the [Django](https://djangoproject.com/) Python Web framework and requires either a [PostgreSQL](https://www.postgresql.org/) or [MySQL](https://www.mysql.com) database backend. It runs as a WSGI service behind your choice of HTTP server.
+
+## Components
+
+<!-- markdownlint-disable no-inline-html heading-style blanks-around-headings-->
+<div class="grid cards" markdown>
```

Comment:
I personally feel like the cards here take up a lot more visual "weight" than the old table without really adding much value or readability:

![image](https://github.com/nautobot/nautobot/assets/5603551/e2b1beec-a08e-461c-afdb-27fc3b1ed4f4)

![image](https://github.com/nautobot/nautobot/assets/5603551/6242d1b4-834d-48d4-9536-dcb417438feb)

What would you think about keeping the table layout and just adding the appropriate icons to it instead?